apps/desktop-wallet/src/modals/SettingsModal/WalletsSettingsSection.tsx

```diff
@@ -108,15 +109,40 @@ const WalletsSettingsSection = () => {
             <Button role="secondary" onClick={lockWallet}>
               {t('Lock current wallet')}
             </Button>
-            <Button role="secondary" onClick={() => setIsQRCodeModalVisible(true)}>
-              {t('Export current wallet')}
-            </Button>
+
+            <ButtonTooltipWrapper
+              data-tooltip-id="default"
+              data-tooltip-content={
+                activeWallet.isPassphraseUsed ? t('To export this wallet use it without a passphrase') : ''
+              }
+            >
+              <Button
+                role="secondary"
+                onClick={() => setIsQRCodeModalVisible(true)}
+                disabled={activeWallet.isPassphraseUsed}
+              >
+                {t('Export current wallet')}
+              </Button>
+            </ButtonTooltipWrapper>
+
             <Button role="secondary" variant="alert" onClick={() => setIsDisplayingSecretModal(true)}>
               {t('Show your secret recovery phrase')}
             </Button>
-            <Button variant="alert" onClick={() => setWalletToRemove(activeWallet as ActiveWallet)}>
-              {t('Remove current wallet')}
-            </Button>
+
+            <ButtonTooltipWrapper
+              data-tooltip-id="default"
+              data-tooltip-content={
+                activeWallet.isPassphraseUsed ? t('To delete this wallet use it without a passphrase') : ''
```

Comment:
I think it makes sense to disable deleting the wallet when it's unlocked with a passphrase. USers might think that they are deleting the "passphrase-enabled" wallet, where in fact they are deleting the 24-words wallet. We don't store any "passphrase-enabled" wallet, so there's nothing to delete.

Reply:
Very valid point!